business-object-model/src/test/java/org/bonitasoft/engine/bdm/BusinessObjectModelConverterTest.java

```diff
@@ -150,7 +150,7 @@ void should_set_current_model_version_when_marshalling() throws Exception {
         final BusinessObjectModel unmarshalledBom = convertor.unmarshall(convertor.marshall(bom));
 
         assertThat(unmarshalledBom.getModelVersion()).isEqualTo(BusinessObjectModel.CURRENT_MODEL_VERSION);
-        assertThat(bom.getProductVersion()).isNotNull().isNotEmpty();
+        assertThat(bom.getProductVersion()).isNull();
```

Comment:
So `productVersion` will not be used anymore? Should we deprecate it then?

Reply:
As I said, the idea is to use it only at runtime from now on.